x-pack/plugins/alerting/README.md

```diff
@@ -97,44 +108,59 @@ The following table describes the properties of the `options` object.
+|services.savedObjectsClient|This is an instance of the saved objects client. This provides the ability to perform CRUD operations on any saved object that lives in the same space as the rule.<br><br>The scope of the saved objects client is tied to the user who created the rule (only when security is enabled).|
+|services.alertInstanceFactory(id)|This [alert factory](#alert-factory) creates alerts and must be used in order to execute actions. The id you give to the alert factory is a unique identifier for the alert.|
 |services.log(tags, [data], [timestamp])|Use this to create server logs. (This is the same function as server.log)|
-|startedAt|The date and time the alert type started execution.|
-|previousStartedAt|The previous date and time the alert type started a successful execution.|
-|params|Parameters for the execution. This is where the parameters you require will be passed in. (example threshold). Use alert type validation to ensure values are set before execution.|
-|state|State returned from previous execution. This is the alert level state. What the executor returns will be serialized and provided here at the next execution.|
-|alertId|The id of this alert.|
-|spaceId|The id of the space of this alert.|
-|namespace|The namespace of the space of this alert; same as spaceId, unless spaceId === 'default', then namespace = undefined.|
-|name|The name of this alert.|
-|tags|The tags associated with this alert.|
-|createdBy|The userid that created this alert.|
-|updatedBy|The userid that last updated this alert.|
+|startedAt|The date and time the rule type started execution.|
+|previousStartedAt|The previous date and time the rule type started a successful execution.|
+|params|Parameters for the execution. This is where the parameters you require will be passed in. (e.g. threshold). Use rule type validation to ensure values are set before execution.|
+|state|State returned from the previous execution. This is the rule level state. What the executor returns will be serialized and provided here at the next execution.|
+|alertId|The id of this rule.|
+|spaceId|The id of the space of this rule.|
+|namespace|The namespace of the space of this rule. This is the same as `spaceId`, unless `spaceId === "default"`, in which case the namespace = `undefined`.|
+|name|The name of this rule. This will eventually be removed in favor of `rule.name`.|
+|tags|The tags associated with this rule. This will eventually be removed in favor of `rule.tags`.|
+|createdBy|The user ID of the user that created this rule. This will eventually be removed in favor of `rule.createdBy`.|
+|updatedBy|The user ID of the user that last updated this rule. This will eventually be removed in favor of `rule.updatedBy`.|
+|rule.name|The name of this rule.|
```

Comment:
These `rule` inputs were added in [this PR](https://github.com/elastic/kibana/pull/99819)